src/quote/swap/adapters/curve/index.ts

```diff
@@ -0,0 +1,53 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { Contract } from '@ethersproject/contracts'
+
+import {
+  SwapQuoteProvider,
+  SwapQuoteRequest,
+  SwapQuote,
+} from 'quote/swap/interfaces'
+import { getRpcProvider } from 'utils/rpc-provider'
+
+import { getSwapData } from './swap-data'
+
+export class CurveSwapQuoteProvider implements SwapQuoteProvider {
```

Comment:
This adapter we need for a few products that specifically use quotes of a Curve pool for ETH/stETH.